pkg/compactor/retention/expiration.go

```diff
@@ -132,6 +133,43 @@ func NewTenantsRetention(l Limits) *TenantsRetention {
 	}
 }
 
+func (tr *TenantsRetention) PolicyFor(userID string, lbs labels.Labels) string {
+	policyStreamMapping := tr.limits.PoliciesStreamMapping(userID)
+	var (
+		matchedRule       *validation.PriorityStream
```

Comment:
nit
```suggestion
		matchedSelector       *validation.PriorityStream
```

Reply:
that change wasn't necessary at the end, I had to rewrite the code